OpenOversight/tests/conftest.py

```diff
@@ -432,6 +432,33 @@ def mockdata(session):
     return add_mockdata(session)
 
 
+@pytest.fixture
+def department(session):
+    department = models.Department(name='Springfield Police Department',
+                                   short_name='SPD', unique_internal_identifier_label='homer_number')
+    session.add(department)
+    session.commit()
+    return department
+
+
+@pytest.fixture
+def department_with_ranks(department, session):
+    for order, rank in enumerate(RANK_CHOICES_1):
+        session.add(models.Job(
+            job_title=rank,
+            order=order,
+            is_sworn_officer=True,
+            department=department
+        ))
+    session.commit()
+    return department
+
+
+@pytest.fixture
+def csv_path(tmp_path):
+    return os.path.join(tmp_path, "file.csv")
```

Comment:
I had to change this to `return os.path.join(str(tmp_path), "file.csv")` or else I would get the following error: `TypeError: join() argument must be str or bytes, not 'PosixPath'`.

Reply:
This sounds like maybe an issue w/ the Python version, which me and @abandoned-prototype hadn't caught before. I'll change it to your version and see if all the tests pass locally and on Travis, and if so I guess we can keep it.

Reply:
Looks like the tests are also failing in travis for python 3.5. @brianmwaters can you add the `str` as dismantl suggested?